app.yaml

```diff
@@ -18,3 +18,7 @@ handlers:
   expiration: 30d
   http_headers:
     Cache-Control: 'public, max-age=2592000'
+
+- url: /.*
```

Comment:
I think you'll need a different URL for main.app. Maybe make this /ajax/.* and put it above /.* so that anything that doesn't fit the /ajax/.* URL pattern "falls through" to the next one. Currently I don't think this handler will ever get called.